### ISSUE TITLE
Can't get slider to work

### ISSUE THREAD

**Issue by merffy88** [kenwheeler/slick#1936]
Hello, thank you so much for your work! However, I must be missing something because my images are  not sliding. I can only see the first image. Any ideas of what i'm missing? Thank you!
<!doctype html>
<html class="no-js" lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Jim Parrish Photography</title>
    <link rel="stylesheet" href="css/foundation.css" />
    <link rel="stylesheet" href="css/styles.css" />
    <script src="js/vendor/modernizr.js"></script>
  <link rel="stylesheet" type="text/css" href="css/slick.css"/>
  <link rel="stylesheet" type="text/css" href="css/slick-theme.css"/>
  </head>
  <body>

    <div class="fixed">
     <nav class="top-bar" data-topbar role="navigation">
  <ul class="title-area">
    <li class="name">
      <h1><a href="index.html" id="mylogo">JIM PARRISH PHOTOGRAPHY</a></h1>
    </li>
     
    <li class="toggle-topbar menu-icon"><a href="#"><span class="mymenuicon">Menu</span></a></li>
  </ul>

  <section  class="top-bar-section">
    
    <ul id="myleft" class="left" >
      <li class="active"><a href="index.html">HOME</a></li>
      <li class="has-dropdown">
        <a href="#">COLLECTIONS</a>
        <ul class="dropdown">
          <li><a href="#">ALL THINGS WILD</a></li>
          <li><a href="#">DESERT SOUTHWEST</a></li>
          <li><a href="#">HAUNTED BY WATERS</a></li>
          <li><a href="#">LEGENDS OF THE FALL</a></li>
          <li><a href="#">LICHENS AS ART</a></li>
          <li><a href="#">NATURAL TREASURES</a></li>
          <li><a href="#">PACIFIC NORTHWEST</a></li>
          <li><a href="#">SECRET GARDEN</a></li>
          <li><a href="#">SUNRISE MOONSET</a></li>
          <li class="active"><a href="#">Active link in dropdown</a></li>
        </ul>
      </li>
      <li><a href="#">ARTIST'S BIO</a></li>
      <li><a href="#">BLOG</a></li>
      <li><a href="#">STORE</a></li>
      <li><a href="#">CONTACT</a></li>

    </ul>
  </section>
</nav>
</div>
<div class="fade">
  <div><img src="img/collections/allthingswild/crane_web.jpg"></div>
  <div><img src="img/collections/desertsouthwest/hoodoos_web.jpg"></div>
  <div><img src="img/collections/hauntedbywaters/painting_web.jpg"></div>
</div>    
    <div class="row">
        <div class="large-12 columns">
        
          <div class="panel">
            <h4>Get in touch!</h4>
                
            <div class="row" "myrow">
              <div class="large-9 columns">
                <p>We'd love to hear from you, you attractive person you.</p>
              </div>
              <div class="large-3 columns">
                <a href="#" class="radius button right">Contact Us</a>
              </div>
            </div>
          </div> 
        </div>
      </div>
     
       
      
      <footer class="row">
        <div class="large-12 columns">
          <hr/>
          <div class="row">
            <div class="large-6 columns">
              <p>© Copyright Jim Parrish Photography</p>
            </div>
            <div class="large-6 columns">
              <ul class="inline-list right">
                <li><a href="#">About</a></li>
                <li><a href="#">Blog</a></li>
                <li><a href="#">Bio</a></li>
              </ul>
            </div>
          </div>
        </div> 
      </footer>
<script src="js/vendor/jquery.js"></script>
<script src="js/foundation.min.js"></script> 
<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
<script type="text/javascript" src="//code.jquery.com/jquery-1.11.0.min.js"></script>
<script type="text/javascript" src="//code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
<script type="text/javascript" src="js/slick.min.js"></script>
<script>
    $(document).foundation();
  </script>
  <script type="text/javascript">
    $(document).ready(function(){
      $('.fade').slick({
      dots: true,
      infinite: true,
      speed: 500,
      fade: true,
      cssEase: 'linear'
 });
    });
  </script>
</body>
</html>

**Comment by KeizerDev** [comment]
Please place your code between the following characters: ```

**Comment by merffy88** [comment]
"'
<!doctype html>
<html class="no-js" lang="en">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>Jim Parrish Photography</title>
    <link rel="stylesheet" href="css/foundation.css" />
    <link rel="stylesheet" href="css/styles.css" />
    <script src="js/vendor/modernizr.js"></script>
  <link rel="stylesheet" type="text/css" href="css/slick.css"/>
  <link rel="stylesheet" type="text/css" href="css/slick-theme.css"/>
  </head>
  <body>

    <div class="fixed">
     <nav class="top-bar" data-topbar role="navigation">
  <ul class="title-area">
    <li class="name">
      <h1><a href="index.html" id="mylogo">JIM PARRISH PHOTOGRAPHY</a></h1>
    </li>
     
    <li class="toggle-topbar menu-icon"><a href="#"><span class="mymenuicon">Menu</span></a></li>
  </ul>

  <section  class="top-bar-section">
    
    <ul id="myleft" class="left" >
      <li class="active"><a href="index.html">HOME</a></li>
      <li class="has-dropdown">
        <a href="#">COLLECTIONS</a>
        <ul class="dropdown">
          <li><a href="#">ALL THINGS WILD</a></li>
          <li><a href="#">DESERT SOUTHWEST</a></li>
          <li><a href="#">HAUNTED BY WATERS</a></li>
          <li><a href="#">LEGENDS OF THE FALL</a></li>
          <li><a href="#">LICHENS AS ART</a></li>
          <li><a href="#">NATURAL TREASURES</a></li>
          <li><a href="#">PACIFIC NORTHWEST</a></li>
          <li><a href="#">SECRET GARDEN</a></li>
          <li><a href="#">SUNRISE MOONSET</a></li>
          <li class="active"><a href="#">Active link in dropdown</a></li>
        </ul>
      </li>
      <li><a href="#">ARTIST'S BIO</a></li>
      <li><a href="#">BLOG</a></li>
      <li><a href="#">STORE</a></li>
      <li><a href="#">CONTACT</a></li>

    </ul>
  </section>
</nav>
</div>
<nav class="breadcrumbs" role="menubar" aria-label="breadcrumbs">
    <li role="menuitem"><a href="index.html">Home</a></li>
    <li role="menuitem"><a href="#">Features</a></li>
    <li role="menuitem" class="unavailable" role="button" aria-disabled="true"><a href="#">Gene Splicing</a></li>
    <li role="menuitem" class="current"><a href="#">Cloning</a></li>
</nav>
<div class="fade">
  <div><img src="img/collections/allthingswild/crane_web.jpg"></div>
  <div><img src="img/collections/desertsouthwest/hoodoos_web.jpg"></div>
  <div><img src="img/collections/hauntedbywaters/painting_web.jpg"></div>
</div>    
    <div class="row">
        <div class="large-12 columns">
        
          <div class="panel">
            <h4>Get in touch!</h4>
                
            <div class="row" "myrow">
              <div class="large-9 columns">
                <p>We'd love to hear from you, you attractive person you.</p>
              </div>
              <div class="large-3 columns">
                <a href="#" class="radius button right">Contact Us</a>
              </div>
            </div>
          </div> 
        </div>
      </div>
     
       
      
      <footer class="row">
        <div class="large-12 columns">
          <hr/>
          <div class="row">
            <div class="large-6 columns">
              <p>© Copyright Jim Parrish Photography</p>
            </div>
            <div class="large-6 columns">
              <ul class="inline-list right">
                <li><a href="#">About</a></li>
                <li><a href="#">Blog</a></li>
                <li><a href="#">Bio</a></li>
              </ul>
            </div>
          </div>
        </div> 
      </footer>
<script src="js/vendor/jquery.js"></script>
<script src="js/foundation.min.js"></script> 
<script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
<script type="text/javascript" src="//code.jquery.com/jquery-1.11.0.min.js"></script>
<script type="text/javascript" src="//code.jquery.com/jquery-migrate-1.2.1.min.js"></script>
<script type="text/javascript" src="js/slick.min.js"></script>
<script>
    $(document).foundation();
  </script>
  <script type="text/javascript">
    $(document).ready(function(){
      $('.fade').slick({
      dots: true,
      infinite: true,
      speed: 500,
      fade: true,
      cssEase: 'linear'
 });
    });
  </script>
</body>
</html>
"'

**Comment by ahmadalfy** [comment]
Please create a jsfiddle demonstrating the problem
